lib/inherited_resources.rb

```diff
@@ -21,18 +21,12 @@ module InheritedResources
   def self.flash_keys=(array)
     Responders::FlashResponder.flash_keys = array
   end
-
-  class Railtie < ::Rails::Engine
-    config.inherited_resources = InheritedResources
-    if config.respond_to?(:app_generators)
-      config.app_generators.scaffold_controller = :inherited_resources_controller
-    else
-      config.generators.scaffold_controller = :inherited_resources_controller
-    end
-  end
 end
 
-class ActionController::Base
+ActiveSupport.on_load(:action_controller) do
```

Comment:
We had to keep this here since we need this behavior in our tests and the railtie is not executed in our tests. I can change it to be a method that include this behavior but I prefer to do in a different PR.